frontend/src/styles/element.scss

```diff
@@ -263,3 +263,10 @@ html {
     background-color: #f5f7fa !important;
     color: var(--el-color-info) !important;
 }
+
+.custom-input-textarea:hover {
+    color: var(--el-color-primary) !important;
+    background-color: var(--el-color-primary-light-9) !important;
+    border-color: var(--el-button-border-color) !important;
+}
+
```

Comment:
The provided CSS modifications look mostly correct from a basic standpoint. However, some improvements could be made:

1. **CSS Selector Consistency**: Ensure that the selector is consistent without extra spaces, like `color: var(--el-button-border-color) !important;` should ideally be just `border-color: var(--el-button-border-color);`.

2. **Variable Names**: It's always good to avoid using generic variable names such as `--el-color-primary`. This can improve maintainability. Consider making them more descriptive if appropriate.

3. **Hover State Styles**: The hover state style is already well-implemented with colors and background changes. No immediate suggestion comes to mind here since it works correctly. 

Here's an optimized version considering best practices:

```css
html {
    background-color: #f5f7fa !important;
    color: var(--main-text-color) !important; /* Replace --main-text-color with a more specific name */
}

.custom-input-textarea:hover {
    /* Simplified styling */
    color: darken(#007bff, 10%) ; /* Use HSLA for better control over lightness */
    background-color: rgba(48,64,96, 0.2);
    border-color: #ccc;
}
```

This optimization improves readability by simplifying unnecessary comments and adjusts the contrast using `darken()` (HSLA function for dynamic adjustments).